.gitignore

```diff
@@ -12,6 +12,7 @@
 /build
 
 # misc
+.vscode
```

Comment:
What does this do? Just curious!

Reply:
Just makes sure VS code local settings don't make it into the development repo.